cadc-access-control-identity/src/test/java/ca/nrc/cadc/ac/ACIdentityManagerTest.java

```diff
@@ -106,6 +107,7 @@ public void testCertOnlyUser() {
             Subject subject = new Subject();
             subject.getPrincipals().add(new X500Principal(dn));
             subject.getPublicCredentials().add(AuthMethod.CERT);
+            System.setProperty(PropertiesReader.CONFIG_DIR_SYSTEM_PROPERTY, "build/resources/test/config");
```

Comment:
if you set a system prop, you really need a try/finally so you can unset it properly after this test; otherwise it can effect other tests in odd ways (because of how junit works, gradle daemon mode, etc).